src/migration/1757630994202-PollVotes.ts

```diff
@@ -0,0 +1,72 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class PollVotes1757630994202 implements MigrationInterface {
+  name = 'PollVotes1757630994202';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            CREATE OR REPLACE FUNCTION user_post_poll_vote_insert_trigger_function()
+            RETURNS TRIGGER AS $$
+            BEGIN
+                IF NEW."pollVoteOptionId" IS NOT NULL THEN
+                    UPDATE poll_option SET "numVotes" = "numVotes" + 1 WHERE id = NEW."pollVoteOptionId";
+                    
+                    UPDATE post SET "numPollVotes" = "numPollVotes" + 1 
```

Comment:
This is just the generic total then right? so total amount of votes on a post.

Reply:
Ah no wait this is the individual option count?
Why do you need to weird select, isn't it just pollVoteOptionId?